src/main/kotlin/org/rust/ide/typing/assist/RsSmartEnterProcessor.kt

```diff
@@ -36,6 +39,10 @@ class RsSmartEnterProcessor : SmartEnterProcessorWithFixers() {
             val elementType = each.node.elementType
             when {
                 elementType == LBRACE || elementType == RBRACE -> continue@loop
+                each is RsMatchArm && each.parent is RsMatchBody -> {
+                    val psiFactory = RsPsiFactory(psiFile.project)
+                    each.add(psiFactory.createComma())
```

Comment:
Also, probably it's better to insert text via `editor.document` not to care about psi correctness
See `org.rust.ide.typing.assist.MethodCallFixer` as exmaple